packages/next/client/index.js

```diff
@@ -407,9 +410,19 @@ function AppContainer({ children }) {
       }
     >
       <RouterContext.Provider value={makePublicRouterInstance(router)}>
-        <HeadManagerContext.Provider value={headManager.updateHead}>
-          {children}
-        </HeadManagerContext.Provider>
+        <FallbackContext.Provider
+          value={
+            // Fallback is true iff server-rendered page was a fallback
+            isFallback &&
+            // And iff the router hasn't successfully switched states yet (on
```

Comment:
```suggestion
            // And if the router hasn't successfully switched states yet (on
```

Reply:
This [iffs](https://www.merriam-webster.com/dictionary/iff) were intentional, "if and only if".